### PR TITLE
Add instance information to the environment

### PR DESCRIPTION
**Pull request by gregarndt**

### CODE REVIEW

File: test/integration/env_test.js [hunk 1]
```diff
@@ -1,25 +1,61 @@
-suite('setting env variables', function() {
-  var co = require('co');
-  var testworker = require('../post_task');
+import assert from 'assert';
+import testworker from '../post_task';
```
Comment (walac) [inline test/integration/env_test.js]: hrm, I thought `import` statements were only allowed at the beggining of the file.

Comment (gregarndt) [inline test/integration/env_test.js]: ah shoot, I put those in the wrong spot.  It works, but I didn't mean for them to be there, I'll fix that.

Comment (gregarndt) [inline test/integration/env_test.js]: These are at the beginning now that I look at it.

File: test/integration/header_footer_logs_test.js [hunk 2]
```diff
@@ -1,25 +1,22 @@
...
+        maxRunTime: 5 * 60
       }
     });
 
-    var tcLogs = result.log.match(/\[taskcluster (.*)\](.*)/g);
-    var start = tcLogs[0];
-    var end = tcLogs[tcLogs.length-1];
+    let tcLogs = result.log.match(/\[taskcluster (.*)\](.*)/g);
+    let start = tcLogs[0];
+    let end = tcLogs[tcLogs.length-1];
```
Comment (walac) [inline test/integration/header_footer_logs_test.js]: good to see things moving to ES6

Comment (gregarndt) [inline test/integration/header_footer_logs_test.js]: Yea, I try to update things as I touch them.

**Comment by walac** [comment]
One test failed but it seems unrelated, r+

**Pull request merged**